mattermost-plugin/webapp/src/index.tsx

```diff
@@ -172,7 +172,14 @@ export default class Plugin {
             }
 
             if (rudderKey !== '') {
-                rudderAnalytics.load(rudderKey, rudderUrl)
+                const rudderCfg = {} as {setCookieDomain: string}
```

Comment:
why not  just initialize the object instead of forcing the type and let typescript to directly infer it. WDYT?
```suggestion
                const rudderCfg = {setCookieDomain: ''}
```

Reply:
That was my first version actually, but given that this is a value we pass to a third-party library, I prefer giving `undefined` rather than an empty string when we don't set it.
Passing an empty string would work today because they are doing a pretty loose test (`options && options.setCookieDomain`) but I feel better giving them `undefined`

Reply:
I see your point, thanks for the clarification.